src/doc/integrating_telio.md

```diff
@@ -131,6 +131,76 @@ telio.destroy()!!;
 
 </multi-code>
 
+
+### Feature config
+
+App can deserialize JSON string with feature config before passing to telio.
+
+<multi-code-select></multi-code-select>
+
+<multi-code>
+
+```rust no_run
+use telio::{ffi::*, types::*};
+
+let json_feature_config = "<feature config fetched from API>".to_owned();
+let feature_config = string_to_features(json_feature_config).unwrap();
+
```

Comment:
to make it a bit more clear i would also add something like:
```rust
feature_config.lana = Some(FeatureLana {  event_path: "some/path.db", prod: true });
if let Some(nurse) = &mut feature.nurse {
    nurse.fingerprint = "me";
}
```
as they will need to set these

Reply:
Added